compatimetrics.py

```diff
@@ -0,0 +1,501 @@
+       numeric vector representing results of model prediction
+
+    Returns
+    -------
+    Numeric value
+       DR value of given prediction vectors
+    """
+    count = 0
+    for i in range(len(pred1)):
+        if pred1[i] != pred2[i]:
+            count += 1
+    return count/len(pred1)
+
+def disagreement_postive_ratio(pred1, pred2, y, positive=None):
+    """ Calculates percentage of observations where two classification models
+    predicted different value in relation to originally positive values.
+
+    Parameters
+    ----------
+    pred1 : list, numpy.array, pandas.series
+        numeric vector representing results of model prediction
+    pred2 : list, numpy.array, pandas.series
+        numeric vector representing results of model prediction
+    y: list, numpy.array, pandas.series
+        true values of predicted variable
+    positive: value, optional
+        value that represents a positive observation in variable
+
+    Returns
+    -------
+    Numeric value
+        DPR value of given predictions pair and true values vector
+    """
+    if positive is None:
+        positive = y.squeeze().unique()[1]
+    T = 0
+    T_disagreement = 0
+    for i in range(len(y)):
+        if y[i] == positive:
+            T += 1
+            if pred1[i] != pred2[i]:
+                T_disagreement += 1
+    return T_disagreement/T
+
+def conjunctive_accuracy(pred1, pred2, y):
+    """ Calculates conjunctive accuracy of two prediction vectors
+    and true values of variable.
+
+    Parameters
+    ----------
+    pred1 : list, numpy.array, pandas.series
+        numeric vector representing results of model prediction
+    pred2 : list, numpy.array, pandas.series
+        numeric vector representing results of model prediction
+    y: list, numpy.array, pandas.series
+        true values of predicted variable
+
+    Returns
+    -------
+    Numeric value
+        Conjunctive accuracy value of given predictions pair
+        and true values vector
+    """
+    same = 0
+    for i in range(len(y)):
+        if pred1[i] == pred2[i] and pred2[i] == y[i]:
+            same += 1
+    return same/len(y)
+
+def conjunctive_precission(pred1, pred2, y, positive=None):
+    """ Calculates conjunctive precision of two prediction vectors
+    and true values of variable.
+
+    Parameters
+    ----------
+    pred1 : list, numpy.array, pandas.series
+        numeric vector representing results of model prediction
+    pred2 : list, numpy.array, pandas.series
+        numeric vector representing results of model prediction
+    y: list, numpy.array, pandas.series
+        true values of predicted variable
+    positive: value, optional
+        value that represents a positive observation in variable
+
+    Returns
+    -------
+    Numeric value
+        Conjunctive precision value of given predictions pair
+        and true values vector
+    """
+    if positive is None:
+        positive = y.squeeze().unique()[1]
```

Comment:
Shouldn't there be `is not None`, and some condition what to do when there is none?